.prettierrc.json

```diff
@@ -0,0 +1,4 @@
+{
+  "semi": false,
```

Comment:
nothing worse than a semi colon!

Reply:
are you guys thumb-downing cause you agree, or you hate me now? haha

Reply:
LOL, I'm one of those Crockford troglodytes who likes semi-colons haha